config.json

```diff
@@ -14,6 +14,20 @@
     "indent_style": "space",
     "indent_size": 4
   },
+  "files": {
+    "solution": [
+      "src/lib.rs"
```

Comment:
It would be nice if we could capture `src/**/*.rs`, but that depends on configlet correctly supporting both recursive and nonrecursive globbing. As mentioned [here](https://github.com/exercism/docs/pull/58#issuecomment-780789579). 

If that isn't currently supported, then these patterns look right to me.